### PR TITLE
add knative support to helm chart

### PR DESCRIPTION
**Pull request by ilackarms**
Signed-off-by: Scott Weiss <sdw35@cornell.edu>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -220,6 +220,9 @@ bump-helm-version:
 install/kube.yaml: $(shell find install/helm/gloo)
 	helm template install/helm/gloo --namespace gloo-system > $@
 
+install/kube-knative.yaml: $(shell find install/helm/gloo)
+	helm template install/helm/gloo --namespace gloo-system --values install/helm/gloo/values-knative.yaml > $@
+
```
Comment (rickducott) [inline Makefile]: Could also start publishing this in the build, just add to RELEASE_BINARIES variable and it should get uploaded as an asset (maybe we change that name to RELEASE_ASSETS)

File: install/helm/gloo/values.yaml [hunk 2]
```diff
@@ -4,10 +4,17 @@ rbac:
   create: true
 
 settings:
+  integrations:
+    knative:
+      enabled: true
+    proxy:
+      image: soloio/gloo-envoy-wrapper:0.5.0
```
Comment (rickducott) [inline install/helm/gloo/values.yaml]: How is this image tag supposed to get updated? 0.5.0 is out of date (any fixed constant will be quickly out of date)

File: install/helm/gloo/templates/5-discovery-deployment.yaml [hunk 3]
```diff
@@ -20,5 +20,5 @@ spec:
       - image: {{ .Values.deployment.discovery.image }}
         imagePullPolicy: {{ .Values.deployment.imagePullPolicy }}
         name: discovery
-        args: ["--namespace", "{{.Release.Namespace}}" ]
+        args: ["--namespace={{.Release.Namespace}}" ]
```
Comment (ilackarms) [inline install/helm/gloo/templates/5-discovery-deployment.yaml]: for some reason this fixed an issue with discovery going into a crash loop @yuval-k @rickducott

File: install/helm/gloo/values.yaml (hunk 2 above)
Comment (rickducott) [reply install/helm/gloo/values.yaml]: followed up on slack, going to leave this as is for this PR and then i'll take the task of getting the makefile to generate values.yaml at build time

**Pull request merged**